modules/acs-requirements.adoc

```diff
@@ -42,9 +42,4 @@ However, you can use another storage type if you do not have SSDs available.
 Use the `helm version` command to verify the version of Helm you have installed.
 * The {ocp} CLI (`oc`).
 * You must have the required permissions to configure deployments in the Central cluster.
-* You must have access to the Red Hat Container Registry. See link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication] for information about downloading images from `registry.redhat.io`.
-+
-[NOTE]
-====
-If you are not a Red Hat customer and purchased the StackRox Kubernetes Security Platform before the acquisition, you can use StackRox’s container registry at `stackrox.io`. Contact mailto:support@stackrox.com[support@stackrox.com] to enable access to the registry.
-====
+* You must have access to the Red Hat Container Registry. See link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication] for information about downloading images from `registry.redhat.io`.
```

Comment:
When introducing inline links, please use For more information about [1]

[1] https://docs.engineering.redhat.com/display/CCSENABLE/Inline+links